jupyterlab_automl/src/components/model_properties.tsx

```diff
@@ -1,221 +1,35 @@
-                          {option.columnName}
-                        </TableCell>
-                        <TableCell>{option.dataType}</TableCell>
-                      </TableRow>
-                    ))}
-                  </TableBody>
-                </Table>
-              </Box>
-            </Collapse>
-          </TableCell>
-        </TableRow>
-      </React.Fragment>
-    );
-  }
-}
-
 export class ModelProperties extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
       hasLoaded: false,
       isLoading: false,
-      modelDetails: [],
-      transformationOptions: [],
       open: false,
+      pipeline: undefined,
     };
   }
 
   async componentDidMount() {
-    this.getPipeline();
-  }
-
-  private getBudget(milliNodeHours: number): string {
-    return (milliNodeHours / 1000).toString() + ' node hours';
-  }
-
-  private getElapsedTime(totalSeconds: number): string {
-    let minutes = Math.floor(totalSeconds / 60);
-    if (minutes > 60) {
-      const hours = Math.floor(minutes / 60);
-      minutes = minutes - hours * 60;
-      const seconds = totalSeconds - hours * 60 * 60 - minutes * 60;
-      return hours + 'hr ' + minutes + ' min ' + seconds + ' sec';
-    } else {
-      const seconds = totalSeconds - minutes * 60;
-      return minutes + ' min ' + seconds + ' sec';
-    }
-  }
-
-  private createData(key: string, val: any) {
-    return { key, val };
-  }
-
-  private getModelDetails(pipeline: Pipeline): any[] {
-    const modelId = this.props.model.id.split('/');
-    const modelDetails = [
-      this.createData('ID', modelId[modelId.length - 1]),
-      this.createData('Region', 'us-central1'),
-    ];
-    for (let i = 0; i < properties.length; i++) {
-      if (pipeline[properties[i]['name']]) {
-        if (
-          properties[i]['name'] === 'trainBudgetMilliNodeHours' ||
-          properties[i]['name'] === 'budgetMilliNodeHours'
-        ) {
-          modelDetails.push(
-            this.createData(
-              properties[i]['label'],
-              this.getBudget(pipeline[properties[i]['name']])
-            )
-          );
-        } else if (properties[i]['name'] === 'elapsedTime') {
-          modelDetails.push(
-            this.createData(
-              properties[i]['label'],
-              this.getElapsedTime(pipeline[properties[i]['name']])
-            )
-          );
-        } else if (properties[i]['name'] === 'createTime') {
-          modelDetails.push(
-            this.createData(
-              properties[i]['label'],
-              pipeline[properties[i]['name']].toLocaleString()
-            )
-          );
-        } else {
-          modelDetails.push(
-            this.createData(
-              properties[i]['label'],
-              pipeline[properties[i]['name']]
-            )
-          );
-        }
-      }
-    }
-    return modelDetails;
+    this.getModelDetails(this.props.pipeline);
```

Comment:
should this be `this.getPipeline()` because `this.getModelDetails()` does not exist in this scope?